chaco/plots/scatterplot_1d.py

```diff
@@ -32,7 +32,9 @@ class ScatterPlot1D(Base1DPlot):
     marker = MarkerTrait
 
     # The pixel size of the marker, not including the thickness of the outline.
-    marker_size = Float(4.0)
+    marker_size = Union(Float, Int,
```

Comment:
The `Float` trait should already accept integers, so I'm not sure that it adds much to accept `Int` here, especially given that the marker size is conceptually a float here.

Reply:
Yeah, I found that Float trait can accomodate int as well, just cahnged.